doppler/assets/logs/doppler.yaml

```diff
@@ -0,0 +1,57 @@
+id: doppler
+metric_id: doppler
+backend_only: false
+facets:
+  - groups:
+      - User
+    name: User Name
+    path: usr.name
+    source: log
+  - groups:
+      - User
+    name: User Email
+    path: usr.email
+    source: log
+  - facetType: list
+    groups:
+      - Event
+    name: Type
+    path: type
+    source: log
+pipeline:
+  type: pipeline
+  name: Doppler
+  enabled: true
+  filter:
+    query: "source:doppler"
+  processors:
+    - type: date-remapper
+      name: Define `createdAt` as the official date of the log
+      enabled: true
+      sources:
+        - createdAt
+    - type: message-remapper
+      name: Define `text` as the official message of the log
+      enabled: true
+      sources:
+        - text
+    - type: attribute-remapper
+      name: Map `user.name` to `usr.name`
+      enabled: true
+      sources:
+        - user.name
+      sourceType: attribute
+      target: usr.name
+      targetType: attribute
+      preserveSource: true
```

Comment:
`preserveSource` should not be set to true, it will duplicate the property, we can simply keep the remapping one to false. Unless you have another good reason to do?
Same for usr.email.